tests/test_unitNormMap.py

```diff
@@ -26,21 +27,30 @@ def test_UnitNormMapBasics(self):
             self.checkPersistence(unitnormmap)
 
             indata = np.array([
-                [1.0, 2.0, -6.0, 30.0, 1.0],
-                [3.0, 99.0, -5.0, 21.0, 0.0],
-                [-5.0, 3.0, -7.0, 37.0, 0.0],
-                [7.0, -23.0, -3.0, 45.0, 0.0],
+                [fullCenter[0], 1.0, 2.0, -6.0, 30.0, 1.0],
+                [fullCenter[1], 3.0, 99.0, -5.0, 21.0, 0.0],
+                [fullCenter[2], -5.0, 3.0, -7.0, 37.0, 0.0],
             ], dtype=float)[0:nin]
             self.checkRoundTrip(unitnormmap, indata)
 
             outdata = unitnormmap.applyForward(indata)
             norm = outdata[-1]
 
+            # the first input point is at the center, so the expected output
+            # [Nan, Nan, ..., Nan, 0]
+            pred_out_at_center = [np.nan] * nin
```

Comment:
Try to be consistent in your naming style; I see a mixture of camel-case, underscores, and undelimited names for variables.